src/bootstrap/mk/Makefile.in

```diff
@@ -52,6 +52,36 @@ check-aux:
 		src/tools/cargo \
 		src/tools/cargotest \
 		$(BOOTSTRAP_ARGS)
+	# Run standard library tests in Miri.
+	# We use a 64bit little-endian and a 32bit big-endian target for max coverage.
+	$(Q)BOOTSTRAP_SKIP_TARGET_SANITY=1 \
+		$(BOOTSTRAP) miri --stage 2 \
+		--target x86_64-unknown-linux-gnu,mips-unknown-linux-gnu \
+		library/core \
+		library/alloc \
+		--no-doc
+	# Some doctests have intentional memory leaks.
+	# Also, they work only on the host.
+	$(Q)MIRIFLAGS="-Zmiri-ignore-leaks -Zmiri-disable-isolation" \
+		$(BOOTSTRAP) miri --stage 2 \
+		library/core \
+		library/alloc \
+		--doc
+	# In `std` we cannot test everything.
+	$(Q)MIRIFLAGS="-Zmiri-disable-isolation" BOOTSTRAP_SKIP_TARGET_SANITY=1 \
+		$(BOOTSTRAP) miri --stage 2 library/std \
+		--no-doc -- \
```

Comment:
I didn't add mips-unknown-linux-gnu here as I think std has much less code that depends on target or endianess, so the extra 10min CI time does not seem worth it.